app/models/tax_svc.rb

```diff
@@ -27,15 +29,37 @@ def get_tax(request_hash)
   end
 
   def cancel_tax(request_hash)
-    if tax_calculation_enabled?
-      log(__method__, request_hash)
-      res = response('cancel', request_hash)
-      logger.debug res
-      JSON.parse(res.body)['CancelTaxResult']
+    log(__method__, request_hash)
+    res = response('cancel', request_hash)
+    logger.debug res
+    response = JSON.parse(res.body)['CancelTaxResult']
+
+    if response['ResultCode'] != 'Success'
+      logger.info_and_debug("Avatax Error: Order ##{response['Messages'][0]['Details']}", response)
+    end
+
+    response
+  rescue => e
+    logger.debug e, 'Error in Cancel Tax'
+    'Error in Cancel Tax'
```

Comment:
See again the custom error from Eric:

https://github.com/boomerdigital/musicmagpie/blob/master/lib/magpie/base.rb#L5-L6